.github/workflows/ephemeral-env.yml

```diff
@@ -28,6 +29,22 @@ jobs:
           const result = pattern.exec(context.payload.comment.body)
           return result === null ? 'noop' : result[1]
 
+    - name: Eval comment body for feature flags
+      uses: actions/github-script@v3
+      id: eval-feature-flags
+      with:
+        script: |
+          const pattern = /FEATURE_(\w+)=(\w+)/g;
+          let results = [];
+          [...context.payload.comment.body.matchAll(pattern)].forEach(match => {
+            const config = {
+              name: `SUPERSET_FEATURE_${match[1]}`,
```

Comment:
Why do we need to prefix these with `SUPERSET_`?

Reply:
Namespacing and convention with other superset env vars.